completions-compute-type.txt

```diff
@@ -19,3 +19,4 @@ RightConcat
 Substract
 SubString
 Translate
+XML2Csv
```

Comment:
Should be either *Xml2Csv* or *XML2CSV* to be consistent.

As it's *Json2Csv*, I guess it's best to have *Xml2Csv*

Reply:
Done